neonvm/controllers/config.go

```diff
@@ -22,6 +24,10 @@ type ReconcilerConfig struct {
 	// This field is passed to neonvm-runner as the `-qemu-disk-cache-settings` arg, and is directly
 	// used in setting up the VM disks via QEMU's `-drive` flag.
 	QEMUDiskCacheSettings string
+
+	// ReconcileFailureInterval is the interval after which a VM will be marked as failing if it has
+	// not reconciled successfully.
```

Comment:
Strictly speaking, if the last reconcile was a success and there just haven't been any more it'll still be considered successful, right?
(should we somehow mark the start of reconciling so that if a reconcile operation is sufficiently long, that _also_ gets marked as stuck?)